lib/roles_conversion.rb

```diff
@@ -1,65 +1,67 @@
 class RolesConversion
```

Comment:
I'm fine with the changes here, just curious as to why you switched from class methods to instance methods?